packages/ui/.env.example

```
@@ -2,5 +2,6 @@ NEXT_PUBLIC_APP_NAME="Sui Oracle Market"
 NEXT_PUBLIC_APP_DESCRIPTION="Full-Stack Sui Oracle Market"
 NEXT_PUBLIC_LOCALNET_CONTRACT_PACKAGE_ID=""
 NEXT_PUBLIC_LOCALNET_SHOP_ID=""
+NEXT_PUBLIC_LOCALNET_PYTH_STATE_ID=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)